### PR TITLE
FAT-167 Initial setup for mod-patron-blocks

### PR DESCRIPTION
**Pull request by roman-barannyk**
This PR is first attempt to create Karate tests for mod-patron-blocks. It features basic CRUD tests for limits API.

Resolves [FAT-167](https://issues.folio.org/browse/FAT-167)

### CODE REVIEW

File: mod-patron-blocks/src/test/java/org/folio/OwnersApiTest.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package org.folio;
+
+import org.folio.testrail.AbstractTestRailIntegrationTest;
+import org.folio.testrail.config.TestModuleConfiguration;
+import org.folio.testrail.services.TestRailIntegrationService;
+import org.junit.jupiter.api.Test;
+
+class LimitsApiTest extends AbstractTestRailIntegrationTest {
```
Comment (OleksandrVidinieiev) [inline mod-patron-blocks/src/test/java/org/folio/OwnersApiTest.java:8]: File name is `OwnersApiTest.java`

File: mod-patron-blocks/src/test/java/org/folio/OwnersApiTest.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  private static final String TEST_BASE_PATH = "classpath:domain/mod-patron-blocks/patron-block-limits/";
+  private static final String TEST_SUITE_NAME = "mod-patron-blocks";
+  private static final long TEST_SECTION_ID = 11064L;
+  private static final long TEST_SUITE_ID = 708L;
```
Comment (OleksandrVidinieiev) [inline mod-patron-blocks/src/test/java/org/folio/OwnersApiTest.java:12]: These values are module-specific

**Comment by OleksandrVidinieiev** [comment]
You should also declare a new module in `pom.xml`. See [this example](https://github.com/folio-org/folio-integration-tests/pull/121/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8).

File: mod-patron-blocks/pom.xml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    <dependencies>
+        <dependency>
+            <groupId>org.folio</groupId>
+            <artifactId>common</artifactId>
+            <version>1.0-SNAPSHOT</version>
+        </dependency>
+        <dependency>
+            <groupId>org.folio</groupId>
+            <artifactId>testrail-integration</artifactId>
+            <version>1.0-SNAPSHOT</version>
```
Comment (alexanderkurash) [inline mod-patron-blocks/pom.xml]: Remove dependency

File: mod-patron-blocks/pom.xml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    <properties>
+        <cucumber.reporting.version>3.8.0</cucumber.reporting.version>
+        <testrail-api-java-client.version>2.0.1</testrail-api-java-client.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.folio</groupId>
+            <artifactId>common</artifactId>
+            <version>1.0-SNAPSHOT</version>
```
Comment (alexanderkurash) [inline mod-patron-blocks/pom.xml]: Check if we need this dependency

**Review by alexanderkurash — APPROVED** [review]

**Pull request merged**